Demo/UI Tests/Venmo UI Tests/Venmo_UITests.swift

```diff
@@ -22,11 +22,11 @@ class Venmo_UITests: XCTestCase {
         demoApp.buttons["Venmo (custom button)"].tap()
     }
     
-    func testTokenizeVenmo_whenSignInSuccessfulWithPaymentContext_returnsToApp() {
+    func testTokenizeVenmo_whenSignInSuccessfulWithPaymentContext_returnsNonce() {
```

Comment:
We updated this test in [this PR with the following comment](https://github.com/braintree/braintree_ios/pull/807/commits/c24079ad1279ce6a7d825f8c8801d6826571001d#r848898247). Seems like we should be fine to revert this since we are updating the payment context ID to now match the merchant ID from the client token.